src/app/components/Item/ItemTableRow.jsx

```diff
@@ -42,10 +42,10 @@ const ItemTableRow = ({ item, bibId, getRecord, searchKeywords }) => {
 
   return (
     <tr className={item.availability}>
-      <td>{item.location}</td>
-      <td>{itemCallNumber}</td>
-      <td>{itemRequestBtn}</td>
-      <td>{item.accessMessage.prefLabel}</td>
+      <td>{item.location || ' '}</td>
+      <td>{itemCallNumber || ' '}</td>
```

Comment:
There's a difference between `''` and `' '`. It's an intentional empty space rather than an empty string. I think it just needs to be tested first though to make sure it's correct.